### PR TITLE
Account List - Apply the user currency choice

### PR DESCRIPTION
**Pull request by jpdup**
WIP - It needs more clean up

### CODE REVIEW

File: Source/WPF/MyMoney/View Selectors/AccountsControl.xaml [hunk 1]
```diff
@@ -10,77 +10,83 @@
...
+    <UserControl.Resources>
 
-    <DataTemplate DataType="{x:Type local:AccountSectionHeader}">
-      <Border BorderThickness="0 0 0 1" BorderBrush="{DynamicResource SectionHeaderBorder}" Margin="0 10 0 0">
-        <Grid>
-          <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="Auto" />
-          </Grid.ColumnDefinitions>
+        <DataTemplate DataType="{x:Type local:AccountSectionHeader}">
```
Comment (clovett) [inline Source/WPF/MyMoney/View Selectors/AccountsControl.xaml:15]: I use an indent level of 2 in XAML files... might be nice to standardize.

**Review by clovett — APPROVED** [review]
Looks great, good to go, the only thing I might add later is to make all this optional - in my case where everything is already USD the accounts view looks a bit redundant & noisy.

**Comment by jpdup** [comment]
I will look at the crash and add some null check.
I'll add the optional rendering for your use case Chris

**Comment by jpdup** [comment]
I pushed a fix for the null check and implemented the optional settings for Chris use case.

**Comment by andreva7** [comment]
I like the flags on the left. It seems they should represent the native currency of the account. Right now mine all show up as US flags, no  matter if the account is US or CDN

**Pull request merged**